data/ai/micro_ais/cas/ca_protect_unit_attack.lua

```diff
@@ -48,7 +48,7 @@ function ca_protect_unit_attack:evaluation(cfg)
                     if counter_damage_table[str] then  -- If so, use saved value
                         max_counter_damage = max_counter_damage + counter_damage_table[str]
                     else  -- if not, calculate it and save value
-                        -- Go thru all weapons, as "best weapon" might be different later on
+                        -- Go through all weapons, as "best weapon" might be different later on
```

Comment:
This change seems pointless, as "thru" is pretty accepted now in an informal context.